scripts/create-relocatable-package.py

```diff
@@ -159,6 +159,7 @@ def ldd(executable):
 ar.add('seastar/scripts')
 ar.add('seastar/dpdk/usertools')
 ar.add('install.sh')
+ar.add('scylla_post_install.sh', arcname="dist/common/scripts/scylla_post_install.sh")
 ar.add('scripts/relocate_python_scripts.py', arcname='relocate_python_scripts.py')
```

Comment:
Why are you copying it twice? It's already copied as part of dist/.

Reply:
It doesn't live inside dist/, however, but at the top level.

I can move it inside dist/ if you prefer, but my reasoning is that this file would better be laid in the source tree together with install.sh.

However once it is installed into the system, my reasoning went, it would be better off at /usr/lib/scylla together with the other scripts

Reply:
For now I will just add a comment for this explaining why this is the case so I can force push the branch.

If you really want this moved I will move tomorrow.